### PR TITLE
clients/besu: add support for engine API tests

### PR DESCRIPTION
**Pull request by jflo**
Besu now supports both http/websockets as well as jwt auth/unauth on a single port. Defaults to 8551 like all of its friends.

### CODE REVIEW

File: clients/besu/besu.sh [hunk 1]
```diff
@@ -124,25 +124,31 @@ else
...
 if [ "$HIVE_GRAPHQL_ENABLED" == "" ]; then
     RPCFLAGS="$RPCFLAGS --rpc-http-enabled --rpc-http-api=ETH,NET,WEB3,ADMIN --rpc-http-host=0.0.0.0"
 else
-    RPCFLAGS="$RPCFLAGS --rpc-http-port=8550" # work around duplicate port error
     RPCFLAGS="$RPCFLAGS --graphql-http-enabled --graphql-http-host=0.0.0.0 --graphql-http-port=8545"
 fi
 
 # Enable WebSocket.
-RPCFLAGS="$RPCFLAGS --rpc-ws-enabled --rpc-ws-api=ETH,NET,WEB3,ADMIN --rpc-ws-host=0.0.0.0"
+#RPCFLAGS="$RPCFLAGS --rpc-ws-enabled --rpc-ws-api=ETH,NET,WEB3,ADMIN --rpc-ws-host=0.0.0.0"
```
Comment (fjl) [inline clients/besu/besu.sh:145]: No WebSocket anymore?

Comment (jflo) [reply clients/besu/besu.sh:145]: Websocket is still in place for the "main" stack. Only the engine api is on a combined http stack that upgrades from http to websockets.

**Review by holiman — APPROVED** [review]

File: clients/besu/besu.sh (hunk 1 above)
Comment (fjl) [reply clients/besu/besu.sh:145]: Ahh, so `--rpc-ws-enabled` is not needed anymore to enable it? I was just wondering why you commented out this flag here.

Comment (fjl) [reply clients/besu/besu.sh:145]: I have tested this, and WebSocket is no longer enabled on besu with this change.

**Pull request merged**